### ISSUE TITLE
Set layer visiblity in Leaflet

### ISSUE THREAD

**Issue by Jeroen-GroeneBij** [tomchadwin/qgis2web#690]
Hi Tom,

Using qgis 2.18.23 and qgis2web 2.34 I noticed the export (Leaflet) doesn't use scale based layer visibility.
Scale based label visibility does work (topic #104), but layer visibility doesn't.

Any hope for a solution?

**Comment by tomchadwin** [comment]
Testing with QGIS 2.18.24 and qgis2web 2.34.0, scale-dependent layer visibility is working for me. Can you give some more specific details so we can try to identify where the issue lies? I'll close his for now, but do add comments, and I'll reopen once we can define the issue better.

**Comment by Jeroen-GroeneBij** [comment]
The map contains several layers, the top layer (point layer) uses labels with scale dependent label visibility. 
Below are polygon layers and point layers, some of them having scale dependent layer visibility. At least, that is what I thought.

I have just found the problem: I have set a rule based Style with scale dependent visibility, that doesn't get exported.
Setting scale dependent visibility in Layer properties does get exported and functions as expected.

It would be nice though to have rule based Style scale dependent visibility. That way I can filter out specific objects within one layer. It does work perfect in qgis.
